purchase_sale_link_by_origin/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+* When users cancel a Purchase Order/ Sale Order, the smart button Purchase on module Sales and Sale on module Purchase don't disappears.
```

Comment:
```suggestion
By default since [Odoo 14.0](https://github.com/odoo/odoo/commit/5a1645a8f8f3560eb778da90b6160b322ce0722e#diff-867f19b2ba78aa5cfaeeb601d4a04cbcbe64e389a8d38939be06c1e547d2934a), PO and SO are linked by their order lines.

This module also link them by the PO's Origin field, to cover more cases. For example:
- if a user cancels a PO, by default the link would have been broken; now it won't;
- or if a user manually defines or updates the Origin field of a PO, it will be taken into account.
```